examples/arithmetic_test.rho

```diff
@@ -0,0 +1,14 @@
+// -25, false, true, -0.4
+
+new arithmetic_test in {
+    contract arithmetic_test(rtn) = {
```

Comment:
It looks like this test doesn't actually *do* anything.  Do you `(show-code (compile ...))` or something to check that it did the right thing?